data/human/human missions.txt

```diff
@@ -10199,3 +10199,1327 @@ mission "Stone to Hope: Thanks"
 			`	He returns to tinkering with the repulsor but continues, "There's no money or reward, but a good deed is its own reward, isn't it?`
 			`	"At least that's what I believe," he says more to himself than to you.`
 				decline
+
+
+mission "Tavish and Kormak 1"
+	name "Tavish and Kormak"
+	description "A pair of academics named Tavish and Kormak claim they are about to make a monumental discovery on <origin>. Take them to <destination> to pick up some mining tools. They can't pay you."
+	source "New India"
+	destination "Clark"
+	passengers 2
+	to offer
+		random < 60
+	on offer
+		conversation
+			`You make your way through the makeshift spaceport, breathing in the thick humid air and feeling the relentless glare of New India's twin suns. The muddy streets are filled with a constant bustling stream of humans, animals, and vehicles fighting either to unload the raw materials of this world, or to load up on the precious goods needed to tame it.`
+			`	There's a small uncrowded cafe and bakery slightly off the beaten path. It seems like a nice place to get some respite from the crowds and the sun.`
+			choice
+				`	(Take a seat in the cafe.)`
+					goto seat
+				`	(Head back to your ship.)`
+
+			`	You see no reason to stick around this rudimentary spaceport, so you make your way back to your ship.`
+				defer
+
```

Comment:
For whatever reason, these extra spaces sometimes cause issues ingame. I found once where the `on complete` didn't fire and once where the label didn't work:
![image](https://github.com/user-attachments/assets/5d70febf-6824-4b66-8c84-e72816dedd69)

They probably aren't breaking _everywhere_, but probably better to get rid of them just to be sure and to align with the structure of most other missions.
```suggestion
```